Test.C

```diff
@@ -293,12 +293,17 @@ int main(){
         // increase amount in index amount if things are bought
         // decrease credits if things are bought
         char storeSelect;
+
         do{
-            printf("Would you like to buy equipment? y/N: ");
+            printf("Would you like to buy equipment? y/N: \n");
+            printf("Press Y to open the store or ENTER to continue: ");
             scanf(" %c", &storeSelect);
             if (storeSelect == 'Y' || storeSelect == 'y'){
                 equipmentStore(&player, equipmentStock);
             }
+            else if (storeSelect == '\r') { //tried \r \n ASCII 10 ASCII 13 ASCII 0x0A
+                break;
+            }
```

Comment:
This showcases what you attempted well, but is not actually functional, so lets comment it out.
```suggestion
            /* attempt at having enter actually skip question without a character input
            else if (storeSelect == '\r') { //tried \r \n ASCII 10 ASCII 13 ASCII 0x0A
                break;
            }
            */
```